src/uct/api/uct.h

```diff
@@ -910,7 +910,8 @@ enum uct_iface_feature {
     UCT_IFACE_FEATURE_AMO64 = UCS_BIT(4), /**< Request 64-bit atomic
                                                operations support */
     UCT_IFACE_FEATURE_TAG   = UCS_BIT(5), /**< Request tag matching offload support */
-    UCT_IFACE_FEATURE_LAST  = UCS_BIT(6)  /**< Used to determine the number
+    UCT_IFACE_FEATURE_FR    = UCS_BIT(6), /**< Request flush remote support */
```

Comment:
UCT_IFACE_FEATURE_FLUSH_REMOTE

Reply:
This is like a quiet() op?  Is there doc to say what happens if the request can't be honoured?

Reply:
Second UCT_IFACE_FEATURE_FLUSH_REMOTE

Reply:
> UCT_IFACE_FEATURE_FLUSH_REMOTE

ok, renamed


> This is like a quiet() op? Is there doc to say what happens if the request can't be honoured?

no, this is fence op implementation. in our plans add such feature into all faces/transports where such operation is required (for now IB only), on other ifaces this flag is just ignored because flush(remote) op will produce single flush operation due to strong ordering provided by transport itself

Reply:
@tonycurtis if a new interface that is created with this feature flag, and exposes PUT_ZCOPY/BCOPY/SHORT support flags (from uct_iface_query), then the interface must support it.